browserslist

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```

Comment:
couldn't get the page loaded on IE11, but maybe its my noobness

Reply:
I didn't get a chance to test ie11, I'll make sure to do that.

Reply:
Got it working with IE11 with a couple more changes, will be pushed momentarily.